app/views/search/_reverb_retirement_modal.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="modal fade sitetour" id="reverbRetirementModal" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <a href="#" class="close modal-close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></a>
+        <h2 class="modal-title" id="reverbRetirementModalLabel">Reverb is Retiring - Welcome to Earthdata Search</h2>
+      </div>
+      <div class="modal-body modal-body-welcome">
+        <p>Reverb is retiring.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
```

Comment:
Can you change this text to...

> You have been redirected to Earthdata Search because Reverb is being retired on January 1st, 2018. Earthdata Search is replacing Reverb. If you would like to go return to Reverb for the time being, you may do so by clicking "Return to Reverb" below. Or you can choose to use Earthdata Search for now. At any point you may opt to return back to Reverb. We will remind you periodically that Reverb is retiring as its retirement date draws closer. If you would like to take a tour of Earthdata Search's main features, you may do so after choosing to stay on Earthdata Search. If you would like a more in-depth demonstration of Earthdata Search, [watch this recorded webinar](https://www.youtube.com/watch?v=eJFHMestpCo).

Reply:
@mreese84 Done :)